www/testlog.php

```diff
@@ -7,55 +7,75 @@
 
 use WebPageTest\Util;
 
-$host = Util::getSetting('host');
-$protocol = $request_context->getUrlProtocol();
+// why are you even here
+if ($userIsBot || Util::getSetting('disableTestlog')) {
+    header('HTTP/1.0 403 Forbidden');
```

Comment:
can we instead throw a Forbidden exception here?

These get handled here: https://github.com/WPO-Foundation/webpagetest/blob/master/www/common.inc#L41


I like giving 404s back to bots and people who don't know much better. We can always log this stuff in our exception handler

Reply:
nice, #til